tensorflow/python/framework/config.py

```diff
@@ -18,10 +18,22 @@
 from __future__ import division
 from __future__ import print_function
 
+from tensorflow.python import _pywrap_tf32_execution
 from tensorflow.python.eager import context
 from tensorflow.python.util import deprecation
 from tensorflow.python.util.tf_export import tf_export
 
+@tf_export('config.tensor_float_32_execution_allowed')
+def tensor_float32_execution_allowed():
+  """Return True if TensorFloat32 operations are enabled on supported hardware.
+  """
+  return _pywrap_tf32_execution.is_allowed()
+
+@tf_export('config.allow_tensor_float_32_execution')
+def allow_tensor_float_32_execution(allow):
+  """Allow use of tensor float32 on supported hardware.
```

Comment:
Can you give a bit of background on TF32? Maybe link to Nvidia documentation as well. Also describe that the default is off but it may be switched on by default in the future. Also spell it out as TensorFloat-32, as that's what the Nvidia marketing spells it as.

Reply:
Done.